packages/ui/src/components/Designer/Sidebar/index.tsx

```diff
@@ -60,47 +51,32 @@ const Sidebar = (props: SidebarProps) => {
             top: 0,
             right: 0,
             position: 'absolute',
-            background: '#fffffffa',
-            color: '#333',
-            border: '1px solid #eee',
             padding: '0.7rem 1rem',
             overflowY: 'auto',
             fontFamily: "'Open Sans', sans-serif",
-            fontWeight: 400,
-            textAlign: 'left',
             boxSizing: 'border-box',
+            background: token.colorBgLayout,
           }}
         >
-          {getActiveSchemas().length === 0 ? (
-            <ListView {...props} />
-          ) : (
-            <DetailView {...props} activeSchema={getLastActiveSchema()} />
-          )}
+          <div>
+            {getActiveSchemas().length === 0 ? (
+              <ListView {...props} />
+            ) : (
+              <DetailView {...props} activeSchema={getLastActiveSchema()} />
+            )}
+          </div>
           <div
             style={{
               position: 'absolute',
-              width: '100%',
-              left: 0,
-              bottom: '1rem',
-              paddingTop: '1rem',
+              bottom: '1.5rem',
+              marginTop: '1rem',
+              left: '50%',
+              transform: 'translateX(-50%)',
             }}
           >
-            <div style={{ marginBottom: '1rem', borderBottom: '1px solid #e5e5e5' }} />
-            <button
-              type="button"
-              style={{
-                padding: '0.5rem',
-                background: '#18a0fb',
-                border: 'none',
-                borderRadius: 2,
-                cursor: 'pointer',
-                margin: '0 auto',
-                display: 'block',
-              }}
-              onClick={addSchema}
-            >
-              <strong style={{ color: '#fff' }}>{i18n('addNewField')}</strong>
-            </button>
+            <Button type="primary" style={{ fontWeight: 600 }} onClick={addSchema}>
```

Comment:
do you need to override font weights here? shouldn't the primary button style take care of this?